openreview/conference/helpers.py

```diff
@@ -305,3 +305,43 @@ def get_submission_revision_stage(client, request_forum):
         additional_fields=submission_revision_additional_options,
         remove_fields=submission_revision_remove_options,
         only_accepted=only_accepted)
+
+def get_comment_stage(client, request_forum):
+
+    commentary_start_date = request_forum.content.get('commentary_start_date', '').strip()
+    if commentary_start_date:
+        try:
+            commentary_start_date = datetime.datetime.strptime(commentary_start_date, '%Y/%m/%d %H:%M')
+        except ValueError:
+            commentary_start_date = datetime.datetime.strptime(commentary_start_date, '%Y/%m/%d')
+    else:
+        commentary_start_date = None
+
+    commentary_end_date = request_forum.content.get('commentary_end_date', '').strip()
+    if commentary_end_date:
+        try:
+            commentary_end_date = datetime.datetime.strptime(commentary_end_date, '%Y/%m/%d %H:%M')
+        except ValueError:
+            commentary_end_date = datetime.datetime.strptime(commentary_end_date, '%Y/%m/%d')
+    else:
+        commentary_end_date = None
+
+    allow_public_comments = 'Public' in request_forum.content.get('participants', '')
+    anonymous = allow_public_comments and 'non-anonymously' not in request_forum.content.get('participants', '')
```

Comment:
may be we should have two options like:

`Non anonymous public`
`anonymous public`
?

Reply:
Never mind, I saw the two options :)